### PR TITLE
README.md for /test

### PR DESCRIPTION
**Pull request by photodesignch**
README.me for test framework.

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+# Automation test framework for Web Chat by using Direct Line API
+
+> ## Disclaimer : The target audience for this document is towards to Web Chat and Direct Line API contributors. Although Direct Line API is underneath the Bot Framework, but to test bots locally, please use the [Microsoft Bot Emulator](https://github.com/Microsoft/BotFramework-Emulator) instead.
+
+## Overview of the test framework eco-system
```
Comment (eanders-MS) [inline test/README.md]: ecosystem is a single word

File: test/README.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+│     Bot       │                                    │┌───────────────────┐│
+│  interactive  │               Direct Line.ts --->  ││ .server           ││
+│     area      │                                    ││ server_content.ts ││
+│               │         ┌─────────────┐            │├───────────────────┤│
+├───────────────┤         │ Mocha tests │    <---    ││ .client           ││
+│ Chat inputbox │  <--->  │ uitest.js   │            │└───────────────────┘│
+└───────────────┘         └─────────────┘            └─────────────────────┘
+</pre>
+
+The workflow of the mock service actually spins up __Web Chat__ with __Direct Line API__.  While mock service side-loads __commands_map.server__ object which will overwrite the default __Direct Line API__ behaviors with current implement.
```
Comment (eanders-MS) [inline test/README.md]: Can you avoid using `actually` here?

File: test/README.md (hunk 1 above)
Comment (photodesignch) [inline test/README.md]: I realized a couple more spelling errors. I corrected them on the newest commit.

**Comment by danmarshall** [comment]
Hi folks, here's a handy [link to the ReadMe on Chris's fork](https://github.com/photodesignch/BotFramework-WebChat/tree/master/test) so you can see the rendered markdown.

**Pull request closed without merging**